bundles/org.openhab.core.automation/src/main/java/org/openhab/core/automation/internal/module/handler/DateTimeTriggerHandler.java

```diff
@@ -71,6 +73,7 @@ public class DateTimeTriggerHandler extends BaseTriggerModuleHandler
     private final @Nullable EventFilter eventFilter;
     private String cronExpression = CronAdjuster.REBOOT;
     private Boolean timeOnly = false;
+    private Long offset = 0L;
```

Comment:
Would a [Duration](https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/time/Duration.html) make sense here?

Reply:
The Configuration / web / rulesdsl only supports limited data types: STRING, DECIMAL, etc. Besides, the granularity of cron is only down to seconds, whereas a Duration can represent milli/nano seconds.